src/tests/Common/testenvironment.proj

```diff
@@ -96,8 +96,7 @@
     <TestEnvironment Include="jitstress2_tiered" JitStress="2" TieredCompilation="1" />
     <TestEnvironment Include="jitstress_isas_incompletehwintrinsic" EnableIncompleteISAClass="1"  />
     <TestEnvironment Include="jitstress_isas_nohwintrinsic" EnableHWIntrinsic="0" />
-    <TestEnvironment Include="jitstress_isas_nohwintrinsic_nosimd" EnableHWIntrinsic="0" FeatureSIMD="0" />
-    <TestEnvironment Include="jitstress_isas_nosimd" FeatureSIMD="0" />
+    <TestEnvironment Include="jitstress_isas_nohwintrinsic" EnableHWIntrinsic="0" />
```

Comment:
This one is now identical to the one defined at L98

Reply:
ah, yes.